app/views/commontator/subscriptions_mailer/comment_created.html.erb

```diff
@@ -1,11 +1,7 @@
 <h4>
-  <%= t 'commontator.email.comment_created.body',
-        creator_name: @creator_name, commontable_name: @commontable_name %>
+  <%=@comment_header%>
 </h4>
 
-<%= render partial: 'commontator/comments/body', locals: { comment: @comment } %>
-
 <p>
-  <%= t 'commontator.email.thread_link_html',
-         comment_url: @comment_url, commontable_name: @commontable_name %>
+  <a href="<%=@comment_url %>">Click here</a> to view <%=@practice_name %> comments
```

Comment:
spacing on both of these (add a space to the left of the variable in each)

Reply:
Got it.

Reply:
![image](https://user-images.githubusercontent.com/60527788/80843344-f2161c00-8bb8-11ea-800c-69258463ebc0.png)